operator/src/main/java/oracle/kubernetes/operator/helpers/DomainPresenceInfo.java

```diff
@@ -574,11 +574,12 @@ private void resetFailureCount() {
     retryCount.set(0);
   }
 
-  public int incrementAndGetFailureCount() {
-    return retryCount.incrementAndGet();
+  public void indicateFailure() {
```

Comment:
I have removed the retryCount from DPI in PR 2609 and use the failure count in the domain status instead.

Reply:
Understood.